File: back-end/back-end/Model/Contato.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace back_end.Model
{
    public class Contato
    {
        [Key]
        public int Id { get; set; }
        [Column("nome")]
        public string Nome { get; set; }
        [Column("email")]
        public string Email { get; set; }
        [Column("telefone")]
        public string Telefone { get; set; }
        [Column("usuario_id")]
        [ForeignKey(nameof(Usuario))]
        public int UsuarioId { get; set; }
        public Usuario? Usuario { get; set; }
    }
}

File: front-end/src/Models/Contato.js
export default class Contato{

    constructor(contato){
        contato = contato || {}

        this.id = contato.id;
        this.nome = contato.nome;
        this.email = contato.email;
        this.telefone = contato.telefone;
        this.usuarioId = contato.usuarioId;
    }

    modeloValidoCadastro(){
        return !!(this.email && this.nome && this.telefone);
    }
}